From 4a7df427fe74e05a1827ecd244a3035aa602aafd Mon Sep 17 00:00:00 2001
From: thanhbui <thanh.bui.fz@renesas.com>
Date: Wed, 14 Oct 2020 10:41:04 +0700
Subject: [PATCH 201/475] ARM: dts: r8a7744: Add EthernetMac device node

Define the generic part of the EthernetMac device in R8A7744 (RZ/G1N).

Signed-off-by: thanhbui <thanh.bui.fz@renesas.com>
---
 arch/arm/boot/dts/r8a7744.dtsi | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7744.dtsi b/arch/arm/boot/dts/r8a7744.dtsi
index ad50cd7..caac91f 100644
--- a/arch/arm/boot/dts/r8a7744.dtsi
+++ b/arch/arm/boot/dts/r8a7744.dtsi
@@ -645,6 +645,18 @@
 			dma-channels = <15>;
 		};
 
+		ether: ethernet@ee700000 {
+			compatible = "renesas,ether-r8a7744";
+			reg = <0 0xee700000 0 0x400>;
+			interrupts = <0 162 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&mstp8_clks R8A7744_CLK_ETHER>;
+			power-domains = <&sysc R8A7744_PD_ALWAYS_ON>;
+			phy-mode = "rmii";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "disabled";
+		};
+
 		avb: ethernet@e6800000 {
 			compatible = "renesas,etheravb-r8a7744",
 				     "renesas,etheravb-rcar-gen2";
-- 
2.7.4

